docs/requirements.txt

```diff
@@ -1,6 +1,7 @@
 numpy
 scipy
 astropy
+mpmath
```

Comment:
Given that I just merged the conda doc thing, could you also please update `environment.yml`? Throw a `- mpmath` in there, in the `pip` section as the one on `conda` seems to be the previous 0.19.

Reply:
I don't seem to have an `environment.yml` in my fork. Was this recently merged?